.maestro/sync_flows/create_account.yaml

```diff
@@ -1,6 +1,6 @@
 # Test Definition: https://app.asana.com/0/1205017362573508/1205017364481021
 appId: com.duckduckgo.mobile.android
-name: "ReleaseTest: Users can create an account"
+name: "ReleaseTest: Users can create a sync account"
 tags:
   - releaseTest
```

Comment:
As discussed let's move this one as well to `syncCriticalPathTest`.

decision: to moved all sync tests to `syncCriticalPathTest` until they are stable enough.